docs/RouterTutorial/react-navigation-tab-kr.md

```diff
@@ -1,414 +1,677 @@
+  - `adaptive` - 스크린에 따른 아이콘 및 라벨 조정여부 설정
+  - `keyboardHidesTabBar` - 키보드 사용시 탭바 숨김 여부 설정
+
+  
+
+###### createMaterialBottomTabNavigator
+
+createMaterialBottomTabNavigator을 사용한 예제에서는 아래에서 볼 수 있듯이 shifting을 이용해 해당하는 탭의 라벨만 표시되고 activetintcolor, inactivetintcolor, barstyle을 이용해 배경색 역시 각각 활성화 여부에 따라 다르게 지정해주었습니다.
+
+<img src="./Tab_createMaterialBottomTabNavigator.gif" style="zoom:67%;" />
+
+createMateiralBottomTabNavigator.js*
+
+```jsx
+import React from 'react';
+import {View} from 'react-native';
+import {createAppContainer} from 'react-navigation';
+import {createMaterialBottomTabNavigator} from 'react-navigation-material-bottom-tabs';
+import Icon from 'react-native-vector-icons/Ionicons';
+import HomeScreen from './screens/HomeScreen';
+import ChatScreen from './screens/ChatScreen';
+import SettingsScreen from './screens/SettingsScreen';
+
+const TabNavigator = createMaterialBottomTabNavigator(
+  {
+    Home: {
+      screen: HomeScreen,
+      navigationOptions: {
+        tabBarLabel: 'Home',
+        tabBarIcon: ({tintColor}) => (
+          <View>
+            <Icon style={[{color: tintColor}]} size={25} name={'ios-home'} />
+          </View>
+        ),
+        activeColor: '#DC143C',
+        inactiveColor: '#226557',
+        barStyle: {backgroundColor: '#FFC0CB'},
+      },
     },
-   });
-   
-   
-   ```
-   
-   *App.js*
+    Chat: {
+      screen: ChatScreen,
+      navigationOptions: {
+        tabBarLabel: 'Chat',
+        tabBarIcon: ({tintColor}) => (
+          <View>
+            <Icon
+              style={[{color: tintColor}]}
+              size={25}
+              name={'ios-chatboxes'}
+            />
+          </View>
+        ),
+        activeColor: '#000080',
+        inactiveColor: '#226557',
+        barStyle: {backgroundColor: '#B0C4DE'},
+      },
+    },
+    Settings: {
+      screen: SettingsScreen,
+      navigationOptions: {
+        tabBarLabel: 'Settings',
+        tabBarIcon: ({tintColor}) => (
+          <View>
+            <Icon
+              style={[{color: tintColor}]}
+              size={25}
+              name={'ios-settings'}
+            />
+          </View>
+        ),
+        activeColor: '#006400',
+        inactiveColor: '#226557',
+        barStyle: {backgroundColor: '#8FBC8F'},
+      },
+    },
+  },
+  {
+    initialRouteName: 'Home',
+    shifting: true,
+  },
+);
+
+export default createAppContainer(TabNavigator);
+
+```
+
+- `shifting` - 3개 이상의 탭이 존재할 경우, 활성화 된 탭의 라벨만 show 여부 설정
+- `labeled` - 라벨 show 여부 설정
+- `activeColor` - 활성화된 탭의 라벨과 아이콘 색깔
+- `inactiveColor` - 비활성화된 탭의 라벨과 아이콘 색깔
+- `barStyle` - tab bar 스타일 설정
+- `initialRouteName` - 로딩시 초기 탭 라우터 설정
+
+
+
+
+
+###### createMaterialTopTabNavigator
+
+createMaterialTopTabNavigator을 사용한 예제에서는 다른 예제들과는 다르게 탭에 icon만 위치해있는것을 볼 수 있습니다. 이는 showIcon, showLabel 의 boolean값을 설정함으로써 show 여부를 결정할 수 있습니다.  또한, indicator style을 통해 활성화된 탭을 가르키는 바의 스타일을 저장할 수 있고 탭을 누를 때 역시 presscolor을 통한 잠깐의 색깔 변경을 나타낼 수 있습니다.
```

Comment:
`indicator style` -> `indicatorStyle`
`presscolor` -> pressColor`
(for consistency)